static/app/components/replays/replayPlayer.tsx

```diff
@@ -14,28 +14,22 @@ interface Props {
   className?: string;
 }
 
-type Dimensions = {height: number; width: number};
-type RootElem = null | HTMLDivElement;
+function BasePlayerRoot({className}: Props) {
+  const {isFullscreen} = useFullscreen();
+
+  // In fullscreen we need to consider the max-height that the player is able
+  // to full up, on a page that scrolls we only consider the max-width.
+  const fixedHeight = isFullscreen;
+
+  const {initRoot, dimensions: videoDimensions, fastForwardSpeed} = useReplayContext();
 
-type RootProps = {
-  fastForwardSpeed: number;
-  flexibleHeight: boolean;
-  initRoot: (root: RootElem) => void;
-  videoDimensions: Dimensions;
-  className?: string;
-};
-
-function BasePlayerRoot({
-  className,
-  fastForwardSpeed,
-  flexibleHeight,
-  initRoot,
-  videoDimensions,
-}: RootProps) {
   const windowEl = useRef<HTMLDivElement>(null);
   const viewEl = useRef<HTMLDivElement>(null);
 
-  const [windowDimensions, setWindowDimensions] = useState<Dimensions>();
+  const [windowDimensions, setWindowDimensions] = useState({
+    width: 0,
+    height: 0,
+  });
```

Comment:
Was able to remove the `type Dimensions` declaration by setting defaults here instead of passing in the generic 👍